### ISSUE TITLE
Where does the theme lightbox come from?

### ISSUE THREAD

**Issue by tomasts248** [fancyapps/fancybox#2501]
Hi,
I mean this theme options:
![153455_07082020](https://user-images.githubusercontent.com/32283036/89681800-a368d400-d8c3-11ea-8ae1-b0f2e47117da.jpg)
May I ask if that theme buttons, border etc  was created by you or inspired used from somewhere else. I ask because I really love it.

**Comment by fancyapps** [comment]
Hi,

Thanks, I designed it, and it's just an upgraded design of the original Fancybox (the original design was inspired by Apple). The most difficult thing was designing the close button. In fact, I spend several weeks searching for the perfect result. At first I wanted to create using vectors, but I couldn't get the natural look I was looking for. After many unsuccessful experiments, I learned that the best result is (especially for inner shadows) when drawn by hand, so it's a PNG file. Sometimes when browsing the internet, I notice that closing button when someone has "borrowed" and that makes me feel nostalgic and a bit proud, too.
